### PR TITLE
renamed variables for module from p to m

### PR DESCRIPTION
**Pull request by shivadbhavsar**
Updated variable names according to #1193

### CODE REVIEW

File: src/targets/gpu/schedule_model.cpp [hunk 1]
```diff
@@ -77,28 +77,28 @@ MIGRAPHX_REGISTER_OP(wait_event)
 MIGRAPHX_REGISTER_OP(set_stream)
 
 std::size_t schedule_model::concurrency() const { return streams; }
-void schedule_model::sched(module& p, instruction_ref ins, std::size_t n) const
```
Comment (kahmed10) [inline src/targets/gpu/schedule_model.cpp:80]: The clang-format error check looks to be unrelated to the changes you made to this file, but I think you should update it anyways. We must have missed the change when we upgraded versions awhile back.

**Review by kahmed10 — APPROVED** [review]

**Comment by umangyadav** [comment]
I find couple of more places where change from`module& p` to `module& m` is required. 

See 
`src/analyze_stream.cpp`,  `src/include/analyze_stream.hpp`,  


In `src/rewrite_rnn.cpp` and `src/rewrite_pooling.cpp` and their header files,  It uses `module& prog` instead of `module& m`  we can change that too for uniformity.

**Comment by umangyadav** [comment]
I find couple of more places where change from`module& p` to `module& m` is required. 

See 
`src/analyze_stream.cpp`,  `src/include/analyze_stream.hpp`,  


In `src/rewrite_rnn.cpp` and `src/rewrite_pooling.cpp` and their header files,  It uses `module& prog` instead of `module& m`  we can change that too for uniformity.

**Comment by shivadbhavsar** [comment]
> I find couple of more places where change from`module& p` to `module& m` is required.
> 
> See `src/analyze_stream.cpp`, `src/include/analyze_stream.hpp`,
> 
> In `src/rewrite_rnn.cpp` and `src/rewrite_pooling.cpp` and their header files, It uses `module& prog` instead of `module& m` we can change that too for uniformity.

analyze_streams was left unchanged as the name "m" was already used for the stream_model parameter. Is there a commonly used variable name for stream_model objects?

**Comment by umangyadav** [comment]
> > I find couple of more places where change from`module& p` to `module& m` is required.
> > See `src/analyze_stream.cpp`, `src/include/analyze_stream.hpp`,
> > In `src/rewrite_rnn.cpp` and `src/rewrite_pooling.cpp` and their header files, It uses `module& prog` instead of `module& m` we can change that too for uniformity.
> 
> analyze_streams was left unchanged as the name "m" was already used for the stream_model parameter. Is there a commonly used variable name for stream_model objects?

I don't think we use `stream_model` variable frequently.  I see it being used in those `analyze_stream` files only. It would be better to rename module var to `m` and `stream_model` to something else i.e. `strm` or `strmm`.

**Review by umangyadav — APPROVED** [review]

**Pull request merged**